shared/dev/dumb-sheet/render.desktop.js

```diff
@@ -54,29 +56,42 @@ class DumbSheetRender extends Component<void, Props, any> {
     this.refs.filterInput.setValue(nextProps.dumbFilter)
   }
 
-  render () {
-    const parts = this.props.dumbFilter.toLowerCase().split(':')
-    let numItemsLeftWeCanShow = 10
-    let filter = parts.join(':')
-
-    if (parts.length > 1) {
-      try {
-        numItemsLeftWeCanShow = parseInt(parts[parts.length - 1], 10)
-        filter = parts.slice(0, parts.length - 1).join(':')
-      } catch (_) { }
+  _filterToParams () {
+    let filter = this.props.dumbFilter.toLowerCase()
+    let numItemsLeftWeCanShowMax = 10
+
+    const itemsMatch = filter.match(/(.*):(\d+)/)
```

Comment:
:sheep: Should this be `/^(.*):(\d+)$/`? Otherwise this could match "foo:10:50" as `numItemsLeftWeCanShowMax = 10`.